docs/proposals/27-project-group.md

```diff
@@ -0,0 +1,138 @@
+
+Such a ProjectGroup manifest looks like this:
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: ProjectGroup
+metadata:
+  name: group1
+spec:
+  namespace: group1-projectgroup
+  projects:
+    - Project1
+    - Project2
+```
+
+The main idea behind `ProjectGroup`s is that they, like projects, have an own namespace.
+Every resource that is shareable (at first only `SeedBinding`s), which is in this namespace, will be shared to all `Project`s in this ProjectGroup.
+
+This sharing mechanism will be achieved by copying the objects from the project group namespace to the namespaces of the projects that belong to the project group.
+
+In the future, more objects can be copied like this or in a similar way with e.g. filtering to allow the sharing of secrets within a project group.
+
+It will not be possible to add any arbitrary project to a project group. Only certain people should be allowed to add projects to a project group.
+
+For this, a new role will be introduced in project groups, that will allow them to add projects, in which they are admin in, to the project group.
+
+Furthermore will it not be possible to add projects to a project group which do not exist.
+
+The `.spec.namespace` field will behave equally as the project `.spec.namespace` field.
+
+#### SeedBinding
+
+To enable the functionality of having exclusive or "private" seeds, a mechanism will be introduced that makes it possible to schedule shoots to specific seeds exclusively to one or more `Project`s.
+
+Seed bindings will be considered during the scheduling of a shoot, in a way that if a shoot belongs to a project with a seed binding, a seed that is targeted by the seed binding will be selected for scheduling.
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: SeedBinding
+metadata:
+  name: project1-privateseed-aws-1
+  namespace: project1-projectgroup
+spec:
+  taintSeed: false
+  seedRef: project1-aws-ha1
+  seedSelector:
+    region: eu-west1
+
+```
+
+If `.spec.taintSeed` is set to `true`, the specified seed(s) will be tainted, so that no other projects outside the project group can schedule shoots on them.
+
+Such tainting bindings will only be allowed to be created by Gardener operators, so that not any arbitrary user can exclusively bind seeds.
+
+Seed bindings will only work in either project or project group namespaces.
+
+As mentioned in [ProjectGroup](#projectgroup), when a seed binding is present in a project group namespace, it will be copied to the respective project namespace.
+
+It will also be possible to allow seed bindings to be in a project namespace, in which it becomes available for a singular project.
+
+The `.spec.seedRef` field specifies which Seed is bound to the project group / project.
+
+Alternatively to such a specific seed, it will be possible to select one or more seeds by a label selector defined in `.spec.seedSelector`.
+
+Either `.spec.seedRef` and `.spec.seedSelector` will only be usable exclusively.
```

Comment:
Do we really want a direct seed reference? The seed selector can be used to express the same thing and it allows to transparently scale in case more than one seed is required.

What is the benefit of having a direct seed reference and how should the scale-up migration work?

Reply:
Yeah, you are totally right about that :D
I removed the direct seed reference